nixos/modules/services/monitoring/mimir.nix

```diff
@@ -56,6 +56,7 @@ in {
         ProtectSystem = "full";
         DevicePolicy = "closed";
         NoNewPrivileges = true;
+        WorkingDirectory = "/var/lib/mimir";
```

Comment:
Out of curiosity why did you need this ?

Reply:
From the commit message:

> Mimir writes files relative to its working directory.
> With this option less paths have to be configured.